lib/tbot/spiffe/trust_bundle_cache.go

```diff
@@ -47,13 +50,21 @@ type BundleSet struct {
 	// (the name of the SPIFFEFederation resource) and excludes the spiffe://
 	// prefix.
 	Federated map[string]*spiffebundle.Bundle
+	stale     chan struct{}
```

Comment:
I'm curious: is the change from a subscription model to a "getter/stale" model motivated just by making the interface a bit easier to use, or is there something else I'm missing?